src/inject.js

```diff
@@ -48,9 +48,9 @@ const stringHandlers = {
     // an array of objects and strings.
     fontFamily: function fontFamily(val) {
         if (Array.isArray(val)) {
-            return val.map(fontFamily).join(",");
+            return uniquify(val.map(fontFamily)).join(",");
```

Comment:
Correct, and I'm happy to revert those last 2 commits which add this behavior if we don't want it.